src/App.jsx

```diff
@@ -0,0 +1,26 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Home from "./pages/Home/Home";
+import Items from "./pages/Items/Items";
+import Header from "./components/Header/Header";
+import "./styles/reset.css";
+import "./styles/global.css";
+import "./App.css";
+import Login from "./pages/Auth/Login";
+import SignUp from "./pages/Auth/SignUp";
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route element={<Header></Header>}>
```

Comment:
```suggestion
        <Route element={<Header />}>
```
Self closing으로 쓰셔도 됩니다!